File: Module.Frontend.TPM/Client/model/tpm/btl/HistoricalBTL.js
Ext.define('App.model.tpm.btl.HistoricalBTL', {
    extend: 'Ext.data.Model',
    idProperty: '_Id',
    breezeEntityType: 'BTL',
    fields: [
        { name: '_Id', type: 'string', hidden: true },
        { name: '_ObjectId', hidden: true },
        { name: '_User', type: 'string', isDefault: true },
        { name: '_Role', type: 'string', isDefault: true },
        { name: '_EditDate', type: 'date', isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: '_Operation', type: 'string', isDefault: true },
        { name: 'EventId', hidden: true, isDefault: true },
        { name: 'EventName', type: 'string', isDefault: true },
        { name: 'Number', type: 'int', hidden: false, isDefault: true },
        { name: 'PlanBTLTotal', type: 'float', hidden: false, isDefault: true },
        { name: 'ActualBTLTotal', type: 'float', hidden: false, isDefault: true },
        { name: 'StartDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: 'EndDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: 'InvoiceNumber', type: 'string', hidden: false, isDefault: true },
        { name: 'Number', type: 'int', hidden: false, isDefault: true }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'HistoricalBTLs',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    }
});

File: Module.Persist.TPM/Model/History/HistoricalBTL.cs
using Core.History;
using Module.Persist.TPM.Model.TPM;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Module.Persist.TPM.Model.History
{
    [AssociatedWith(typeof(BTL))]
    public class HistoricalBTL : BaseHistoricalEntity<System.Guid>
    {
        public int Number { get; set; }
        public DateTimeOffset? StartDate { get; set; }
        public DateTimeOffset? EndDate { get; set; }
        public double? PlanBTLTotal { get; set; }
        public double? ActualBTLTotal { get; set; }
        public string InvoiceNumber { get; set; }
        public string EventName { get; set; }
        public string EventDescription { get; set; }
    }
}
